### PR TITLE
add inclusterconfig filter for commenting kube-proxy configmap

### LINKED ISSUE #1127: [Question] How to install openyurt on multiple control-plane and worker nodes?
What happened:

I am installing a kubeadm based openyurt cluster, thanks for the great doc. 

In the documentation it says under [Join Node](https://openyurt.io/docs/installation/yurtadm-join) that "You should only install node components of OpenYurt on nodes that have already been joined in the Kubernetes cluster."

What does that mean exactly?

I want my setup to be ha. for this I need three control plane nodes.
Should the openyurt components described under [manually setup](https://openyurt.io/docs/installation/manually-setup) run on them? Currently these are scheduled on already joined worker nodes. 

How do I handle worker nodes running general things like argocd/fleet/monitoring etc.? Can I just leave them as they are, or do they also need to be modified as described under  [Join Node](https://openyurt.io/docs/installation/yurtadm-join)?

Thanks a lot

Environment:

OpenYurt version: 1.1.0
Kubernetes version (use kubectl version): 1.22.17

/kind question

### PR DESCRIPTION
**Pull request by rambohe-ch**
/kind feature

#### What this PR does / why we need it:
1. add inclusterconfig filter for commenting kubeconfig setting in `kube-system/kube-proxy` configmap, in order to make kube-proxy on nodes with yurthub(like edge nodes) to use `InClusterConfig` to access kube-apiserver.  at the same time, kube-proxy on master nodes without yurthub can use kubeconfig to access kube-apiserver.

2. delete kube-proxy configure in `yurtctl init test` command.

#### Which issue(s) this PR fixes:

Fixes #1127 

#### Does this PR introduce a user-facing change?

```release-note

```

### CODE REVIEW

**Comment by rambohe-ch** [comment]
@Congrool @LaurenceLiZhixin PTAL

File: pkg/yurtctl/cmd/yurttest/kindinit/init.go [hunk 1]
```diff
@@ -411,9 +411,9 @@ func (ki *Initializer) configureAddons() error {
 		return err
 	}
 
-	if err := ki.ConfigureKubeProxyAddon(); err != nil {
-		return err
-	}
+	//if err := ki.ConfigureKubeProxyAddon(); err != nil {
```
Comment (LaurenceLiZhixin) [inline pkg/yurtctl/cmd/yurttest/kindinit/init.go:414]: I guess this function should be deleted.

Comment (rambohe-ch) [reply pkg/yurtctl/cmd/yurttest/kindinit/init.go]: fixed

**Review by Congrool — COMMENTED** [review]
/lgtm

**Pull request merged**